templates/panda_server-httpd-FastCGI.conf.rpmnew.template

```diff
@@ -25,9 +26,12 @@ MinSpareThreads        ${PANDA_SERVER_CONF_MIN_WORKERS}
 ServerLimit            ${PANDA_SERVER_CONF_MAX_WORKERS}
 MaxSpareThreads        ${PANDA_SERVER_CONF_MAX_WORKERS}
 MaxRequestWorkers      ${PANDA_SERVER_CONF_MAX_WORKERS}
-MaxConnectionsPerChild 2000
+MaxConnectionsPerChild 100
```

Comment:
It is good to set a non-zero value to avoid huge memory consumption by memory leak, but isn't 100 too small? Generally, it is a few thousand to prevent busy child process spawning.

Reply:
I will test some bigger number to check.

Reply:
The MaxConnectionsPerChild with 2000 is ok. So I set it 2000. I also replaced some fixed number to environment variables. It will be easier to tune it.